container_files/tutorials/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,924 @@
+       "      <th>Logistic regression</th>\n",
+       "      <td>99</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Deep belief network</th>\n",
+       "      <td>98</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Restricted boltzmann machines</th>\n",
+       "      <td>124</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Hopfield network</th>\n",
+       "      <td>53</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Naive bayes classifier</th>\n",
+       "      <td>165</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Boltzmann machine</th>\n",
+       "      <td>89</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Autoencoder</th>\n",
+       "      <td>31</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Artificial neural network</th>\n",
+       "      <td>40</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                               count\n",
+       "_rowName                            \n",
+       "Support vector machine           149\n",
+       "Logistic regression               99\n",
+       "Deep belief network               98\n",
+       "Restricted boltzmann machines    124\n",
+       "Hopfield network                  53\n",
+       "Naive bayes classifier           165\n",
+       "Boltzmann machine                 89\n",
+       "Autoencoder                       31\n",
+       "Artificial neural network         40"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mldb.query(\n",
+    "    \"\"\"\n",
+    "    SELECT horizontal_sum({*}) AS count\n",
+    "    FROM ml_concepts\n",
+    "    \"\"\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Taking the transpose of data\n",
+    "There are two ways to take the transpose of a dataset:\n",
+    "    1. using a transposed dataset\n",
+    "    2. using a transpose SQL function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Response [201]>\n"
+     ]
+    }
+   ],
+   "source": [
+    "print mldb.put(\n",
+    "    \"/v1/datasets/transposed_concepts\", \n",
+    "    {\n",
+    "        \"type\": \"transposed\",\n",
+    "        \"params\": \n",
+    "        {\n",
+    "            \"dataset\": {\"id\":\"ml_concepts\"}\n",
+    "        }\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have transposed our data, we can easily see which words are most frequently used in the Machine Learning concept descriptions."
```

Comment:
J'ajouterais un peu plus de viande, parce que la query suivante n'est pas évidente pour un néophyte: Un the following call, we will use the `transposed_concepts` dataset we juste created and join it with the results of doing the same transpose inline.